certchain/ocsp_response.go

```diff
@@ -53,6 +53,9 @@ var DummyOCSPResponse *OCSPResponse = &OCSPResponse{
 func ParseOCSPResponse(bytes []byte) (*OCSPResponse, error) {
 	resp, err := ocsp.ParseResponse(bytes, nil)
 	if err != nil {
+		if string(bytes) == "dummy-ocsp" {
```

Comment:
Can you say `== DummyOCSPResponse.Raw` instead?

Reply:
Changed but it has to be cast to string because I can't compare byte slices, it only allows for nil comparison.

Reply:
Works for me. IIRC, casting bytes to string is cheap (reuses underlying buffer).